plat/allwinner/sun50i_a64/sunxi_power.c

```diff
@@ -119,6 +120,51 @@ static int axp_setbits_8(uint8_t reg, uint8_t set_mask)
 	return rsb_write(AXP803_RT_ADDR, reg, regval);
 }
 
+static bool fdt_handle_regulator(const void *fdt, int node)
```

Comment:
This isn't very descriptive. I suggest something to the effect of `should_enable_regulator`, and a comment in the code that the existence of a phandle means it's used elsewhere in the DT